proctl/proctl_test.go

```diff
@@ -49,7 +50,10 @@ func dataAtAddr(pid int, addr uint64) ([]byte, error) {
 
 func assertNoError(err error, t *testing.T, s string) {
 	if err != nil {
-		t.Fatal(s, ":", err)
+		_, file, line, _ := runtime.Caller(1)
+		fname := filepath.Base(file)
+		fmt.Printf("\t%s:%d: %s : %s\n", fname, line, s, err)
+		t.FailNow()
```

Comment:
I would change this to `t.Fatalf()` and then use your formatted string you're currently passing into Printf. Also, preface it with something like "failed assertion at:" or something along those lines.